src/email/sender.js

```diff
@@ -0,0 +1,36 @@
+const nodemailer = require('nodemailer');
+const path = require('path');
+const ejs = require('ejs');
+require('dotenv').config();
+
+async function sendForgotPasswordEmail(toEmail, resetToken) {
+  const transporter = nodemailer.createTransport({
+    service: 'gmail',
```

Comment:
This should all already be set up in the `mailing/sender.js` file: [https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/main/src/mailing/sender.js#L4-L10](https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/main/src/mailing/sender.js#L4-L10)

We should be writing this new code in that file and just adding this new function. The function should not need to recreate a transporter object and can just use the one created at top-level of the `sender.js` file

Reply:
I have set the code from  email/sender.js in mailing.js . Should I delete  sender.js in email folder?

Reply:
commit ef22adbf4473cd09a28dba0a7a1ae5139cbdae02